lua/gatekeeperc.lua

```diff
@@ -0,0 +1,78 @@
+local ffi = require("ffi")
+
+--Structs
+ffi.cdef[[
+
+struct bp_config {
```

Comment:
Okay. Let's keep the current format for now. But each commit should ONLY add the definitions it needs.